tmt/steps/prepare/install.py

```diff
@@ -0,0 +1,68 @@
+import fmf
+import tmt
+import click
+
+class PrepareInstall(tmt.steps.prepare.PreparePlugin):
+    """
+    Install packages on the guest
+
+    Example config:
+
+    prepare:
+        how: install
+        copr: psss/tmt
+        package: tmt-all
+
+    Use 'order' attribute to select in which order preparation should
+    happen if there are multiple configs. Default order is '50'.
+    Default order of required packages installation is '70'.
+    """
+
+    # Supported methods
+    _methods = [tmt.steps.Method(name='install', doc=__doc__, order=50)]
+
+    @classmethod
+    def options(cls, how=None):
+        """ Prepare command line options """
+        return [
+            click.option(
+                '-p', '--package', metavar='NAME', multiple=True,
+                help='Package name to be installed.'),
+            click.option(
+                '-c', '--copr', metavar='REPO', multiple=True,
+                help='Copr repository to be enabled.')
+            ] + super().options(how)
+
+    def show(self):
+        """ Show provided scripts """
+        super().show(['package', 'copr'])
+
+    def wake(self, data=None):
+        """ Override options and wake up the guest """
+        super().wake(['package', 'copr'])
+
+        # Convert to list if necessary
+        tmt.utils.listify(self.data, split=True, keys=['package', 'copr'])
+
+    def go(self, guest):
+        """ Prepare the guests """
+        super().go()
+
+        # Enable copr repositories
+        coprs = self.get('copr')
+        if coprs:
+            self.debug('Make sure dnf copr plugin is available.')
+            guest.execute(
```

Comment:
this won't work on EPEL6/7 ....

Reply:
I guess there are more issues like this in the code though ....

If we have this broken state, maybe it would be better to refuse test anything < RHEL8 / Fedora?

Reply:
I think we should (at least for now) clearly state that we support Fedora and RHEL8+. Once we have things working for at least for those, we can extend, hm?

Reply:
Let's explicitly mentioned supported distros:
https://github.com/psss/tmt/pull/221